src/Polygon2d.elm

```diff
@@ -261,6 +261,28 @@ convexHull points =
     singleLoop (lower ++ upper)
 
 
+{-| Constructs a regular convex polygon with the specified center point, circumradius and number of sides.
+-}
+regular : Point2d units coordinates -> Quantity Float units -> Int -> Polygon2d units coordinates
+regular center radius sides =
+    let
+        n =
+            toFloat sides
+
+        angle =
+            Angle.turns 1 |> Quantity.divideBy n
+
+        startAngle =
+            Angle.degrees 90 |> Quantity.minus (Angle.degrees 180 |> Quantity.divideBy n)
```

Comment:
Hm not in my testing... But I wonder if I am getting confused with SVG coordinate systems...

Reply:
Right, so by bottom you mean that y = yMax rather than y = yMin?

Reply:
Oh yeah, good point, I was assuming positive Y being up, but SVG has positive Y as down! `elm-2d-drawing` will fix this, though 😈  (all Y coordinates get negated internally when rendering to SVG), so I think I'd like to stick with the +Y-is-up convention in `elm-geometry`.

Reply:
I'm getting confused here. If you check out the test case it looks like this:

```elm
 Polygon2d.regular (Point2d.meters 0.5 0.5) (meters (sqrt 2 / 2)) 4
                    |> Expect.polygon2d
                        (Polygon2d.singleLoop
                            [ Point2d.meters 1 0
                            , Point2d.meters 0 0
                            , Point2d.meters 0 1
                            , Point2d.meters 1 1
                            ]
                        )
```

So do we want it to look like:

```elm

 Polygon2d.regular (Point2d.meters 0.5 0.5) (meters (sqrt 2 / 2)) 4
                    |> Expect.polygon2d
                        (Polygon2d.singleLoop
                            [ Point2d.meters 1 1
                            , Point2d.meters 0 1
                            , Point2d.meters 0 0
                            , Point2d.meters 1 0
                            ]
                        )
```

?

Reply:
OK I finally had some time to look at this and I think the confusion stems largely from the fact that `Polygon2d.singleLoop` normalizes its argument so that the vertices are always in counterclockwise order - so in fact both of the above tests would pass!

I've tweaked the code to my proposed version which has the bottom edge (assuming Y-up) being horizontal, with the start point of the polygon being the right end of the bottom edge and the end point being the left end of the bottom edge. See what you think!

Reply:
LGTM